<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathNewFile</class>
 <widget class="QWidget" name="PathNewFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>800</width>
     <height>530</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>110</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Название файла</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNameNewFile">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>550</y>
     <width>680</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChoosePath">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>590</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
